src/main/scala/com/github/tkawachi/doctest/TestGenResolver.scala

```diff
@@ -31,10 +32,15 @@ object TestGenResolver {
     }
   }
 
+  @deprecated(message = "use findScalaTestVersionFromScalaBinaryVersion", since = "0.9.8")
   def findScalaTestVersion(testClasspath: Classpath, scalaVersion: String): Option[String] = {
-    val scalaBinVersion = scalaVersion.split('.').take(2).mkString(".")
```

Comment:
This old implementation is wrong if milestone or RC scalaVersion.

e.g. `"3.0.0-M1".split('.').take(2).mkString(".")` is `"3.0"` but `scalaBinaryVersion` is `"3.0.0-M1"` when `scalaVersion` is `"3.0.0-M1"`